SIS_import/Dockerfile

```diff
@@ -34,4 +32,6 @@ RUN mkdir /usr/src/app/data
 RUN chown -R root:root /usr/src/app/data
 RUN chmod g+rw /usr/src/app/data
 
+ENV PROCESS=sis_set_url
```

Comment:
Why is this needed?

Reply:
the PROCESS variable is used inside [rclone_setup.sh](https://github.com/tl-its-umich-edu/canvas/blob/master/SIS_import/rclone_setup.sh), where it defaults to value`sis_set_url ` if not set. 

I added it here to demonstrate that this is a configurable ENV variable inside the Dockerfile.